src/Routes.tsx

```diff
@@ -14,12 +14,41 @@ import { CommonRoutesFeatures } from './RoutesWrap';
 const { MainView } = lazily(() => import('~views/Main/Main'));
 const { Reports } = lazily(() => import('~views/Reports/Reports'));
 const { ReportPage } = lazily(() => import('~views/Report/Report'));
+const { AboutPage } = lazily(() => import('~views/About/About'));
 const { BivariateManagerPage } = lazily(
   () => import('~views/BivariateManager/BivariateManager'),
 );
 
 export function RoutedApp() {
   const [{ data: userModel, loading }] = useAtom(userResourceAtom);
+
+  function isFirstTimeVisit() {
+    const firstTimeVisit =
+      userModel &&
+      !loading &&
+      location.pathname == '/' &&
+      location.search == '' &&
+      !localStorage.getItem('landed');
+    if (firstTimeVisit) {
+      localStorage.setItem('landed', 'yes');
```

Comment:
i know that it doesn't matter what string to store there, but 'true'  would follow our previous flags